apps/yarn.lock

```diff
@@ -1306,6 +1306,15 @@
   version "2.0.2"
   resolved "https://registry.yarnpkg.com/@blockly/field-grid-dropdown/-/field-grid-dropdown-2.0.2.tgz#61c531e4082f0424a8afccd9c87ebe5286744158"
   integrity sha512-wA0nTdhgNI4xGPcODOuQhB43LcvDtwWZcorlNtyuTFJcYHidYOyTkC5i99HpiUxo4qdrt0C3OqkLp1gmJPEBXA==
+"@blockly/field-grid-dropdown@~2.0.2":
```

Comment:
i'm confused about this change -- we already have `@blockly/field-grid-dropdown@~2.0.2` directly above this line, and the information looks identical

Reply:
Nice catch - this was likely just a mistake on my part due to a merge conflict. i must have accepted both versions here.